src/site/_includes/content/origin-trials.njk

```diff
@@ -1,6 +1,4 @@
 Origin trials allow you to try new features and give feedback on their
 usability, practicality, and effectiveness to the web standards community. For
-more information, see the
-<a href="https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md" rel="noopener">Origin Trials Guide for Web Developers</a>.
-To sign up for this or another origin trial, visit the
-<a href="https://developers.chrome.com/origintrials/#/trials/active" rel="noopener">registration page</a>.
+more information, see the [Origin Trials Guide for Web Developers](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md).
+To sign up for this or another origin trial, visit the [registration page](https://developers.chrome.com/origintrials/#/trials/active).
```

Comment:
Let's keep the HTML links because they're using [rel="noopener"](https://developers.google.com/web/tools/lighthouse/audits/noopener). So please change this back.

Reply:
Hi @kaycebasques thanks for reaching out, please correct I'm wrong but according to [this article](https://developers.google.com/web/tools/lighthouse/audits/noopener) `rel="noopener"` is only necessary/required if the link has `target="_blank"`

Reply:
Hi, @kaycebasques do you have an update for this one?
I made a new commit and a new reviewer was automatically added, not sure whether that's an issue since you're already reviewing this PR.

Reply:
Ah, I missed that. You're right, thanks @IvanBila.